src/app/api/sumupPayment/sumup.api.ts

```diff
@@ -0,0 +1,39 @@
+import axios from "axios"
+
+type response = {
+  id: string,
+  status: string
+}
+
+const pay_to_email = process.env.PAY_TO_EMAIL
+export async function getCheckoutId(amount: string) {
+  const referenceId = Math.floor(100000 + Math.random() * 900000)
```

Comment:
Maybe here we can use an UUID.

This math is cool but no guarantee no duplicated records